vendor/vendor.json

```diff
@@ -173,4 +173,4 @@
 			"revisionTime": "2015-06-24T11:29:02+01:00"
 		}
 	]
-}
+}
```

Comment:
Yes, ugly, but messing with generated files doesn't really solve this either.
:+1: for solving this upstream.